### DISCUSSION TITLE
SSD NVMe

### DISCUSSION THREAD

**Discussion by ehn29** [PartialVolume/shredos.x86_64#142] [Q&A] [answered]
Is it possible to wipe NVMe SSD with SHREDOS/NWIPE  ?
Test on Toshiba KXG60ZNV

no SDD detected even with command "nvme list"

I use the latest versions of shredos and nwipe (shredos-2021.08.2_23_x86-64_0.34_20221231.img)

**Comment by PartialVolume** [comment]
It should certainly recognise it's there. Can you ALT F2 to bring up the virtual terminal in ShredOS and type lspci and take a photo and post the result. We may be missing a driver.

**Reply by ehn29** [reply]
here is

![20230418_114500](https://user-images.githubusercontent.com/131145362/232739877-030b73f1-3f9d-4c26-8d6a-d8207b3ae451.jpg)
![20230418_113821](https://user-images.githubusercontent.com/131145362/232739891-54949a4b-bb4c-41c8-9dea-3069db4ee470.jpg)

**Reply by PartialVolume** [reply]
The SATA AHCI libata drivers built into ShredOS should support that, however your laptop may not have the raid controller in the correct mode. Is it a ACER laptop by any chance?

Take a look at this as it might be relevant https://community.acer.com/en/discussion/611426/how-to-change-sata-mode-from-optane-without-raid-to-ahci-acer-aspire-7-a715-75g

**Comment by PartialVolume** [comment]
For instance, here's the 82801 SATA controller chip shown in lspci from a HP xW4600 running the SATA emulation in AHCI mode. The mode that the controller has currently activated is shown via the highlighted lspci line showing the 82801 controller in ACHI mode. In the bios of the xW4600 you can switch SATA emulation mode to RAID which is shown in the lspci data. Having said that, on the HP even in RAID mode a SATA drive shows up in ShredOS. Maybe it's different for a SSD.
.
![82801 SATA controller](https://user-images.githubusercontent.com/22084881/232880230-dd6256f5-f991-4ddb-9841-f1550314cfae.png)

**Comment by PartialVolume** [comment]
Re-reading you initial comment I'm not sure it has anything to do with SATA controller as you were talking about the NVME M.2 interface. That may be controlled by Sunrise Point memory controller. As to whether there is a driver in ShredOS that supports that, I've looked and can't find anything specific named Sunrise Point except a pinctrl driver. My own laptop has NVME M.2 and that appears as a memory controller `2c:00.0 Non-Volatile memory controller: Sandisk Corp WD Blue SN570 NVMe SSD` and ShredOS detects that without a problem. So other than something in the BIOS that may be affecting it like NvMe raid option I've not really got any idea why that's not being seen.

**Comment by ehn29** [comment, marked as answer]
My computer is a DELL  and by default the SATA controller is in RAID.
By putting it in AHCI, the SDD is well recognized then.
Thanks for your help

**Reply by PartialVolume** [reply]
That's great, thanks for letting us know. 👍
